Cargo.toml

```diff
@@ -17,7 +17,8 @@ categories = ["game-development", "simulation"]
 all-features = true
 
 [features]
-default = []
+default = ["scenes"]
+scenes = ["heron_core/scenes"]
```

Comment:
also update readme and root crate documentation about the new feature flag.

Why "scene" though. I thought it is more about "mesh" isn't?

Reply:
Hm... Makes sense, actually.

Reply:
I am not sure what would be the best feature name. Maybe `mesh-hull` or `shape-from-mesh`? I am open to suggestion :-)

Reply:
How about `convex-from-mesh`?

Reply:
No, probably `shape-from-mesh` would be better.

Reply:
Or maybe `collision-from-mesh`?

Reply:
Yeah, let's go with `shape-from-mesh`. No need to bikeshed too much. I think that is explicit enough.

Reply:
I renamed it to `shape-from-mesh` first, but on second thought `collision-from-mesh` looks like a more consistent name. So I added a commit for the second rename. Feel free to disagree, I will just revert it.